src/settings/SettingsScreen.js

```diff
@@ -51,13 +53,31 @@ export default function SettingsScreen(props: Props): Node {
           dispatch(setGlobalSettings({ browser: value ? 'embedded' : 'external' }));
         }}
       />
-      <SwitchRow
-        label="Do not mark messages read on scroll"
-        value={doNotMarkMessagesAsRead}
-        onValueChange={value => {
-          dispatch(setGlobalSettings({ doNotMarkMessagesAsRead: value }));
-        }}
-      />
+      <View
+        style={[
+          styles.listItem,
+          {
+            height: 56,
+            justifyContent: 'space-between',
+          },
+        ]}
+      >
+        <Text style={{ fontSize: 15 }}>Mark messages read on scroll</Text>
+        <View>
+          <Picker
+            selectedValue={doNotMarkMessagesAsRead}
+            style={{ width: 150 }}
```

Comment:
Why 150?

Reply:
same as this [comment](https://github.com/zulip/zulip-mobile/pull/5294#discussion_r865996951)